### PR TITLE
OCPBUGS-8299: CronJobs table/details UI doesn't have Suspend indication

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://issues.redhat.com/browse/OCPBUGS-8299

Add spec.suspend to the list and details view for cron jobs.

Screenshot 2023-03-12 at 4 05 30 PM
Screenshot 2023-03-12 at 4 05 16 PM

### CODE REVIEW

File: frontend/public/components/cron-job.tsx [hunk 1]
```diff
@@ -77,12 +79,15 @@ const CronJobTableRow: React.FC<RowFunctionArgs<CronJobKind>> = ({ obj: cronjob
       </TableData>
       <TableData className={tableColumnClasses[2]}>{cronjob.spec.schedule}</TableData>
       <TableData className={tableColumnClasses[3]}>
-        {_.get(cronjob.spec, 'concurrencyPolicy', '-')}
+        {cronjob.spec?.suspend ? i18next.t('public~true') : i18next.t('public~false')}
```
Comment (rhamilto) [inline frontend/public/components/cron-job.tsx:82]: Use `True` and `False` here and below since the CLI output uses capitalization of these values?

Comment (jcaianirh) [reply frontend/public/components/cron-job.tsx]: @rhamilto thanks! updated

**Pull request merged**